src/unix/core.c

```diff
@@ -79,6 +79,10 @@
 # endif
 #endif
 
+#ifdef _AIX
+#include <sys/ioctl.h>
```

Comment:
Is this necessary? It appears to be unconditionally included in L34 above.

Reply:
removed unnecessary include statement